seinfoflow

```diff
@@ -75,11 +78,28 @@ elif args.verbose:
 else:
     logging.basicConfig(level=logging.WARNING, format='%(message)s')
 
+conditionals = None
+if args.conditionals == 'default':
+    conditionals = {}
+elif args.conditionals is not None:
+    conditionals = {}
+    for conditional in args.conditionals.split():
```

Comment:
My preference would be to split on comma, not space, so that it is consistent with other types of lists on the CLI (e.g. `-c` and `-p` in sesearch). That also has the upside of not requiring quoting. Since Booleans can't have spaces in the name, this should be fine.